src/main/java/com/mewna/catnip/entity/impl/user/PresenceImpl.java

```diff
@@ -69,25 +69,25 @@ public void catnip(@Nonnull final Catnip catnip) {
     @Nonnull
     @CheckReturnValue
     public JsonObject asJson() {
-        final JsonObject innerData = new JsonObject()
```

Comment:
Can you not chain the method calls here?

Reply:
sadly not. gotta use JsonObject.builder() if you want chaining

Reply:
Imo that would be preferable to this